webapp/channels/src/components/suggestion/search_channel_with_permissions_provider.test.tsx

```diff
@@ -1,12 +1,11 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import {getState} from 'stores/redux_store';
+import SearchChannelWithPermissionsProvider from 'components/suggestion/search_channel_with_permissions_provider';
 
+import getState from 'stores/redux_store';
```

Comment:
Why do we need to change from `{getState}` to `getState`?

Reply:
<img width="711" alt="Screenshot 2023-05-17 at 9 40 40 PM" src="https://github.com/mattermost/mattermost-server/assets/43727167/9c993560-81cd-4aee-acfa-ebcab7949378">

Because this is complaining

Reply:
Thanks for the explanation

Reply:
I think this is actually the reason the tests are failing. The default export of `stores/redux_store` is an object with a `getState` method. Maybe this would work if it was written
```js
import store from 'stores/redux_store';
const getState = store.getState;
```